bot/exts/evergreen/splitify.py

```diff
@@ -0,0 +1,130 @@
+                top += y_frac
+                bottom += y_frac
+                left = 0
+                right = x_frac
+            else:
+                left += x_frac
+                right += x_frac
+
+        return new_imgs
+
+    @staticmethod
+    def join_images(imgs: list) -> Image:
+        """Stitches all the image squares into a new image."""
+        random.shuffle(imgs)
+        img_sizes = [img.size for img in imgs]
+
+        single_wdith = img_sizes[0][0]
+        single_height = img_sizes[0][1]
+
+        multiplier = int(math.sqrt(len(imgs)))
+
+        total_width = multiplier * single_wdith
+        total_height = multiplier * single_height
+
+        new_image = Image.new('RGBA', (total_width, total_height), (250, 250, 250))
+
+        width_multiplier = 0
+        height = 0
+
+        pieces = math.sqrt(len(imgs))
+
+        for index, img in enumerate(imgs):
+            width = single_wdith * width_multiplier
+
+            new_image.paste(img, (width, height))
+
+            width_multiplier += 1
+
+            if (index + 1) % pieces == 0:
+                width_multiplier = 0
+                height += single_height
+
+        return new_image
+
+    @commands.command(name='splitify')
+    async def splitify_command(self, ctx: commands.Context, squares: int = 16) -> None:
+        """Splits your avatar in x squares, randomizes them and stitches them back into a new image!"""
+        async with ctx.typing():
+            if squares < 1:
+                raise commands.BadArgument('Squares must be a `positive number`:exclamation:')
+
+            if not math.sqrt(squares).is_integer():
+                raise commands.BadArgument('Squares must be a `perfect square`:exclamation:')
+
+            if squares > MAX_SQUARES:
+                raise commands.BadArgument('Number of squares cannot be higher than `193,600` :nerd:')
+
+            author = ctx.author
+            image_bytes = await author.avatar_url.read()
+            avatar = Image.open(BytesIO(image_bytes))
+            avatar = avatar.convert('RGBA').resize((1024, 1024))
+
+            img_squares = self.split_image(avatar, squares)
+            new_img = self.join_images(img_squares)
+
+            bufferedio = BytesIO()
+            new_img.save(bufferedio, format='PNG')
+            bufferedio.seek(0)
+
+            file = discord.File(bufferedio, filename='splitifed.png')
+
+            if squares == 1:
+                title = 'Hooh... that was a lot of work'
+                description = 'I present to you... Yourself!'
+            elif squares == MAX_SQUARES:
+                title = 'Testing the limits I see...'
+                description = 'What a masterpiece. :star:'
+            else:
+                title = 'Your splitified avatar'
+                description = 'Here is your avatar. I think it looks a bit *puzzly*'
+
+            embed = discord.Embed(title=title,
+                                  description=description)
```

Comment:
This is against the style guide we follow, which can be seen [here](https://pythondiscord.com/pages/contributing/style-guide/)

To be more specific, this part:
> In the case where items contained in brackets need to be broken across multiple lines, items should be dropped to a new line after the opening bracket with an additional level of indentation. The closing bracket ends on it's own new line, on the same indentation level as the opening bracket.

```suggestion
            embed = discord.Embed(
              title=title,
              description=description
            )
```

Reply:
Alright, I'll do that.